pages/[entityListType]/[...params].tsx

```diff
@@ -201,7 +201,8 @@ export default EntityDetailPage;
  * @returns catalog prefix.
  */
 function getCatalogPrefix(defaultCatalog: string): string {
-  return defaultCatalog.replace(/\d.*$/, "");
+  //eslint-disable-next-line sonarjs/slow-regex -- catalog numbers should be short and are user provided
+  return defaultCatalog.replace(/\d.*$/, ""); //TODO - are all catalog numbers less than a maximum length? could remove eslint ignore
```

Comment:
@frano-m can we replace the `.*` here with `.{0,5}` or something else with a specified length to clear this error? I don't think leaving it disabled is a big deal but it might be neater